doc/ledger3.texi

```diff
@@ -512,7 +512,7 @@ Ledger will generate:
 
 @smallexample @c output:66E3A2C
 10-Dec-01 Checking balance      Assets:Checking          $ 1,000.00   $ 1,000.00
-                                Equit:Opening Balances  $ -1,000.00            0
+                                Equity:Opening Balances  $ -1,000.00            0
```

Comment:
Hi @thdox,
that's quite odd. At the moment I have no idea or explanation why this may be. Thanks for taking this issue to the list; I will respond there once I've done some more investigation.